include/net/ieee802154_radio.h

```diff
@@ -96,6 +96,16 @@ struct ieee802154_radio_api {
 	/** Get the available amount of Sub-GHz channels */
 	u16_t (*get_subg_channel_count)(struct device *dev);
 #endif /* CONFIG_NET_L2_IEEE802154_SUB_GHZ */
+
+#ifdef CONFIG_NET_L2_OPENTHREAD
+	/** Run an energy detection scan.
+	 * Note: channel must be set prior to request this function.
+	 */
+	int (*ed_scan)(struct device *dev,
+		       u16_t duration,
```

Comment:
I wonder what the duration means here meaning seconds, milliseconds or something else, perhaps document it here?

Reply:
According to the header file from Thread:

/**
 * This method begins the energy scan sequence on the radio.
 *
 * @param[in] aInstance      The OpenThread instance structure.
 * @param[in] aScanChannel   The channel to perform the energy scan on.
 * @param[in] aScanDuration  The duration, in milliseconds, for the channel to be scanned.
 *
 * @retval OT_ERROR_NONE             Successfully started scanning the channel.
 * @retval OT_ERROR_NOT_IMPLEMENTED  The radio doesn't support energy scanning.
 */
otError otPlatRadioEnergyScan(otInstance *aInstance, uint8_t aScanChannel, uint16_t aScanDuration);